napalm/iosxr/iosxr.py

```diff
@@ -2085,6 +2098,15 @@ def traceroute(
         try:
             ipv = IPAddress(destination).version
         except AddrFormatError:
+            try:
+                destination
```

Comment:
What are you trying to test here?

Reply:
If a user passes destination=None or an implementation populates the variable, I can safely attempt to populate the passed destination parameter then include it in the logging message

Reply:
Wouldn't just `if destination is None` work? I've never seen this approach with an empty try-except block...

Reply:
Unfortunately python black doesn't like those so I had to reinvent the wheel here lol.

Reply:
Sorry, that won't do it, please just remove this altogether... that `logger.error` in the form you have now should be fine without this try block.

Reply:
Hey @mirceaulinic new commit is up with this removed so we should be good.

Again thank you for going through this PR!